### ISSUE TITLE
Done talking button (mic off) missing voiceover label

### ISSUE THREAD

**Issue by ricardovice** [rogertalk/roger#1]
> When voiceover is running, and you start talking to someone, the microphone button becomes unlabeled. It is just an unlabeled buttons, and double tapping it does release the microphone but that button needs to be labeled in such a way as to be clear that double tapping it should release the microphone. As it stands, there is no clear way for a voiceover user to understand how to stop talking. :-)
